truffle/contracts/Splitter.sol

```diff
@@ -14,6 +14,9 @@ contract Splitter is Stoppable {
     function splitEth(address receiver1, address receiver2) public _onlyIfRunning payable returns(bool) {
         uint half = msg.value / 2;
         
+        require(balances[receiver1] + half >= balances[receiver1]
+            && balances[receiver2] + half >= balances[receiver2], 'Balances are full. They should be withdrawed');
```

Comment:
It looks like you are concerned with overflow. Overflow/underflow is a good thing to think about. What you are doing is correct, in principle. You can take this opportunity to explore "SafeMath" library from Open Zeppelin. Also an opportunity to explore libraries themselves. Their example has some interesting syntax sugar. 

I didn't mention this because the inputs are constrained by Ether and it isn't conceivable that any of these accounts can have that much money. In cases where it is not possible to reason about constraints, then it is important to confirm that a+b>a and a-b<a, etc.